### ISSUE TITLE
Additional features for Highcharts Accessibility

### ISSUE THREAD

**Issue by kiranpawar-gep** [highcharts/highcharts#17841]
Hi Team,

We want to implement below Additional features for Highcharts Accessibility.

1. when we are navigating with arrow keys through different charts. i.e Bar chart, it should announce Bar series 1 of 2 with 20 bars → At present it is announcing the Bar number on which focus is there.

2. When we are using drill up/down functionality, it should announce correct warning messages. i.e Cannot drill down further

3. For a particular chart i.e bar chart, currently it is announcing Bar name and graphic. It should announce Bar name as a button. Please find attached image(barChart) for your reference.

![barChart](https://user-images.githubusercontent.com/41055561/194809214-3437eb0b-dd57-44c8-b0fd-286ce4466255.png)

**Comment by oysteinmoseng** [comment]
Hi @kiranpawar-gep, thanks for requesting! We have some user research into drill down charts that we will incorporate into Highcharts in the near future.

For now, you can achieve 1) and 3) with the following changes: 
1: Set [accessibility.point.valueDescriptionFormat](https://api.highcharts.com/highcharts/accessibility.point.valueDescriptionFormat) or use the [accessibility.point.descriptionFormatter](https://api.highcharts.com/highcharts/accessibility.point.descriptionFormatter) to change the text for the bars.
3: Add the following code before creating the chart in order to change the graphic roles to buttons:
```javascript
Highcharts.addEvent(Highcharts.Chart, 'render', function() {
  this.series.forEach(s =>
      s.points.forEach(p => p.graphic.element.setAttribute('role', 'button'))
  );
});
```
You can also make additional changes in the above `render` event handler, if desired.
